### PR TITLE
Emphasize that this example works is about advertising which operates in disconnected mode.

### PR DESCRIPTION
**Pull request by pan-**

### CODE REVIEW

**Comment by pan-** [comment]
@iriark01 Could you review this ?

File: BLE_GAPButton/README.md [hunk 1]
```diff
@@ -28,10 +28,14 @@ Building instructions for all samples are in the [main readme](https://github.co
 
 1. Open the BLE scanner on your phone.
 
-1. Find your device.
+1. Find your device and look at the advertisement broadcasted by your device (there is no need to connect to your device).
 
-1. The Service Data field for your device should show the button press count. The starting value is 0.
+1. The Service Data field of the advertisement packet broadcasted by your device reflect the button press count. The starting value is 0.
```
Comment (iriark01) [inline BLE_GAPButton/README.md]: Reflects

File: BLE_GAPButton/README.md [hunk 2]
```diff
@@ -28,10 +28,14 @@ Building instructions for all samples are in the [main readme](https://github.co
...
 
 1. Press the button on the device.
 
-1. The Service Data field value should change.
+1. The Service Data field value of the advertisement packet should change every time you press the button.
+
+## Note
+
+Since broadcasting is not reliable and your phone may scan intermittently, it is possible that your phone miss button updates.
```
Comment (iriark01) [inline BLE_GAPButton/README.md]: .... that your phone will miss button updates

**Comment by iriark01** [comment]
Done, cheers

**Comment by pan-** [comment]
Thanks, I have updated my PR. 
I will merge it now.

**Pull request merged**